pkg/apis/operator/v1alpha1/types.go

```diff
@@ -319,6 +331,14 @@ type SNI struct {
 	DomainPatterns []string `json:"domainPatterns,omitempty"`
 }
 
+// Networking defines networking parameters for the virtual garden cluster.
+type Networking struct {
+	// Services is the CIDR of the service network. This field is immutable.
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Value is immutable"
+	Services string `json:"services"`
```

Comment:
Why does the operator need to know the service cidr? It cannot be about network policies as service IPs are not used for network policies.

Reply:
It is needed by the `kube-apiserver`: https://github.com/gardener/gardener/blob/482da4c4fabce60b14df455c7a84975c0f29fa82/pkg/operation/botanist/component/kubeapiserver/deployment.go#L569